---
layout: page
title: Publications
permalink: /Publications/

tagline: 'peer-reviewed and in preprint.'
---

# For an updated list of publications see [*my Google Scholar*](https://scholar.google.com/citations?user=St7QVnoAAAAJ&hl=en){:target="_blank"} page.
<div style="text-align:center"><span style="font-size:1em;line-height:1.5;background-color:transparent"></span></div>


<div><br>
(in review) Hether, T.D., C.G. Wiench, &amp; P.A. Hohenlohe. Evaluation of novel molecular and analytical tools for estimation of rates of meiotic crossovers and non-crossover gene conversions.&nbsp;</div>
<div><br>
Hether, T.D. 2016. Uplift and erosion of genomic islands with standing genetic variation. biorxiv. doi: http://dx.doi.org/10.1101/057513 (<a href="http://biorxiv.org/content/early/2016/06/09/057513">pdf here</a>)&nbsp;</div>
<div><br>
Hand, B.K., T.D. Hether, R.P. Kovach, C.C. Muhlfeld, S.J. Amish, M.C. Boyer, S.M. O'Rourke, M.R. Miller, W.H. Lowe, P.A. Hohenlohe, &amp; G. Lukart. 2015. Genomics and introgression: Discovery and mapping of thousands of species-diagnostic SNPs using RAD sequencing. Current Zoology 61 (1): 146-154. (<a href="http://www.actazool.org/temp/%7B93E0B782-5086-4EC2-A4D6-DAFDB61DC68A%7D.pdf">pdf here</a>)
<div><br>
Hether, T.D. and P.A. Hohenlohe. 2014. Genetic regulatory network motifs constrain adaptation through curvature in the landscape of mutational (co)variance. Evolution. 68 (4), 950-964. (<a href="http://onlinelibrary.wiley.com/doi/10.1111/evo.12313/abstract?systemMessage=Wiley+Online+Library+will+be+disrupted+on+21st+March+from+10%3A30+GMT+up+to+six+hours+and+from+05%3A30+EDT+up+to+six+hours+for+essential+maintenance.++Apologies+for+the+inconvenience.&amp;userIsAuthenticated=false&amp;deniedAccessCustomisedMessage=">pdf here</a>)</div>
<div><br>
Rosenblum, E.B., B.A. Sarver, J.W. Brown, S. Des Roches, K. M. Hardwick, T.D. Hether, J.M. Eastman, M.W. Pennell, and L.J. Harmon. 2012. Goldilocks meets Santa Rosalia: an ephemeral speciation model explains patterns of diversification across time scales. Evolutionary Biology. 39, number 2, 255-261.&nbsp;</div>
<div><br>
Hether, T.D. and E.A. Hoffman. Machine learning identifies specific habitats associated with genetic connectivity in Hyla squirella. 2012. J. Evolutionary Biology 25, issue 6, 1039-1052.&nbsp;</div>
<div><br>
Degner, J.F., D.M. Silva, T.D. Hether, J.M. Daza, E.A. Hoffman. 2010. Fat frogs, mobile genes: unexpected phylogeographic patterns for the ornate chorus frog (Pseudacris ornata). Molecular Ecology 19, issue 12, 2501-2515.</div>
<div><br>
Jenkins D., Carey M., Czerniewska J., Fletcher J., Hether T., Jones A., Knight S., Knox J., Long T., Mannino M., McGuire M., Riffle A., Segelsky S., Shappell L., Sterner A., Strickler T., Tursi R. 2010 Isolation by distance: 20th century relic or reference standard for 21st century landscape genetics? Ecography 33, issue 2, 315-320.</div>
<div><br>
Hether, T. D. and E. A. Hoffman. Characterization of five dinucleotide and six tetranucleotide polymorphic microsatellite loci for the squirrel treefrog (Hyla squirella). Appeared in Abdoullaye D, Acevedo I, Adebayo AA, et al. 2010. Permanent Genetic Resources added to Molecular Ecology Resources Database 1 August 2009-30 September 2009. Molecular Ecology Resources 10, 232-236.</div>
<div><br>
Degner, J. F., T. D. Hether, and E. A. Hoffman. 2009. Eight novel tetranucleotide and five cross-species dinucleotide microsatellite loci for the ornate chorus frog (Pseudacris ornata). Molecular Ecology Resources 9, 622-624.<br>
 <br>
<br>

</div>






<div class="intro"><br>
  <p>
 <a href="https://www.facebook.com/tyler.hether"><i class="fa fa-facebook"></i></a>
 &nbsp; &nbsp; &nbsp;
 <a href="http://github.com/tylerhether"><i class="fa fa-github"></i></a>
 &nbsp; &nbsp; &nbsp;
 <a href="https://twitter.com/tylerhether"><i class="fa fa-twitter"></i></a>
 &nbsp; &nbsp; &nbsp;
 <a href="https://www.flickr.com/photos/126551691@N02"><i class="fa fa-flickr"></i></a>
 </p>
</div>
